packages/synapse-react-client/src/lib/style/components/_total-query-results.scss

```diff
@@ -32,7 +32,7 @@
   padding-left: 10px;
   display: inline-flex;
   justify-content: center;
-  border-radius: SRC.$button-border-radius;
+  border-radius: 20px;
```

Comment:
Revert styling of selection criteria pills: 
<img width="170" alt="feature_SelectionCriteriaPill" src="https://user-images.githubusercontent.com/26949006/231906982-85049337-0774-48c8-bccb-5cc6db0d2e3a.png">